From 6bda069f21d1a61cc63277f46c34f8512f3713fc Mon Sep 17 00:00:00 2001
From: diannamcallister <dianna.mcallister@sickkids.ca>
Date: Wed, 22 May 2019 12:04:51 -0400
Subject: [PATCH] increased text limit to 65000

---
 dwv.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dwv.js b/dwv.js
index 32c60c6..cdbecb4 100644
--- a/dwv.js
+++ b/dwv.js
@@ -3193,7 +3193,7 @@ dwv.dicom.DicomElementsWrapper = function (dicomElements) {
 dwv.dicom.DicomElementsWrapper.prototype.getElementValueAsString = function ( dicomElement, pretty )
 {
     var str = "";
-    var strLenLimit = 65;
+    var strLenLimit = 65000;
 
     // dafault to pretty output
     if ( typeof pretty === "undefined" ) {
-- 
2.17.1

