--- @appbaseio/reactivesearch/lib/components/date/DateRange.js      2019-06-21 11:52:24.691516390 -0400
+++ @appbaseio/reactivesearch/lib/components/date/DateRange.js  1985-10-26 04:15:00.000000000 -0400
@@ -191,7 +191,7 @@
                                                },
                                                onDayChange: this.handleStartDate,
                                                inputProps: {
+                                                       readOnly: false
-                                                       readOnly: true
                                                },
                                                classNames: {
                                                        container: (0, _helper.getClassName)(this.props.innerClass, 'daypicker-container') || 'DayPickerInput',
@@ -234,7 +234,7 @@
                                                },
                                                onDayChange: this.handleEndDate,
                                                inputProps: {
+                                                       readOnly: false
-                                                       readOnly: true
                                                },
                                                classNames: {
                                                        container: (0, _helper.getClassName)(this.props.innerClass, 'daypicker-container') || 'DayPickerInput',
@@ -431,8 +431,8 @@
        autoFocusEnd: true,
        numberOfMonths: 2,
        placeholder: {
+               start: 'yyyy-mm-dd',
+               end: 'yyyy-mm-dd'
-               start: 'Start date',
-               end: 'End date'
        },
        showClear: true,
        showFilter: true
