diff --git a/datefinder/constants.py b/datefinder/constants.py
index e83830c..820e329 100644
--- a/datefinder/constants.py
+++ b/datefinder/constants.py
@@ -10,8 +10,6 @@ TIMEZONES_PATTERN = "ACDT|ACST|ACT|ACWDT|ACWST|ADDT|ADMT|ADT|AEDT|AEST|AFT|AHDT|
 ## explicit north american timezones that get replaced
 NA_TIMEZONES_PATTERN = "pacific|eastern|mountain|central"
 ALL_TIMEZONES_PATTERN = TIMEZONES_PATTERN + "|" + NA_TIMEZONES_PATTERN
-DELIMITERS_PATTERN = r"[/\:\-\,\s\_\+\@]+"
-
 # Allows for straightforward datestamps e.g 2017, 201712, 20171223. Created with:
 #  YYYYMM_PATTERN = '|'.join(['19\d\d'+'{:0>2}'.format(mon)+'|20\d\d'+'{:0>2}'.format(mon) for mon in range(1, 13)])
 #  YYYYMMDD_PATTERN = '|'.join(['19\d\d'+'{:0>2}'.format(mon)+'[0123]\d|20\d\d'+'{:0>2}'.format(mon)+'[0123]\d' for mon in range(1, 13)])
@@ -35,7 +33,8 @@ UNDELIMITED_STAMPS_PATTERN = "|".join(
 DELIMITERS_PATTERN = r"[/\:\-\,\.\s\_\+\@]+"
 TIME_PERIOD_PATTERN = r"a\.m\.|am|p\.m\.|pm"
 ## can be in date strings but not recognized by dateutils
-EXTRA_TOKENS_PATTERN = r"due|by|on|during|standard|daylight|savings|time|date|dated|of|to|through|between|until|at|day"
+EXTRA_TOKENS_PATTERN = r"awesrdtfyg" #prevents issuesof missing dates with the word Date infront                                                      
+#EXTRA_TOKENS_PATTERN = r"due|by|on|during|standard|daylight|savings|time|date|dated|of|to|through|between|until|at|day"
